style.css

```diff
@@ -427,53 +491,89 @@ img {vertical-align: middle;}
 	height: 0;
 	top: 0;
 	left: 0;
-	background-color: crimson;
+	background-color: #4d96db;
 }
 
 .container:hover .overlayCross{
 	height: 100%;
 	width: 100%;
+	background-color: orange;
 }
 
 .overlayFade{
 	width: 100%;
 	height: 100%;
 	top: 0;
 	left: 0;
-	background-color: lightgreen;
+	background-color: #4c7397;
 }
 /* Headers for Grid 4 & Grid 3 divs (Animals & tips and What to brign text) */
 
 .grid_4_headers {
 	box-sizing: border-box;
 	background-color: white;
 	grid-column: span 14;
-	height: 130px;
+	height: 120px;
+}
+
+.grid_4B_headers {
+	box-sizing: border-box;
+	grid-column: span 14;
+	height: 80px;
+	margin-top: 50px;
+  text-shadow: grey 1px 0 10px;
 }
 
 .grid_3_headers {
 	box-sizing: border-box;
-	background-color: #E3E3E3;
 	height: 170px;
 	grid-column: span 15;
 }
 
 .grid_4_headers h1, .grid_3_headers h1{
-  font-size: clamp(2.5em, 3.5vw, 8em);
+  font-size: clamp(24px, 2.5vw, 36px);
 	text-align: center;
 	width: 100%;
 	position: absolute;
-	color: #555c66;
+	color: #c78007;
+	padding: 20px 0px 0px 0px;
+	font-family: 'Roboto', sans-serif;
 }
 
-.grid_4_headers h2, .grid_3_headers h2 {
+.grid_4B_headers h1{

```

Comment:
Now that I think about it, this font doesn't really work with his header, I will consider changing ()